### PR TITLE
improvement: use AnyStr to keep intellisense for enums but allow forw…

### PR DESCRIPTION
**Pull request by armandobelardo**
…ard compat

### CODE REVIEW

**Review by dsinghvi — APPROVED** [review]

File: generators/python/sdk/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.4] - 2024-03-19
+
+- Improvement: Allow full forward compat with enums while keeping intellisense by unioning enum literals with typing.AnyStr.
```
Comment (dsinghvi) [inline generators/python/sdk/CHANGELOG.md:10]: nit: can we add the before and after signature in a python code snippet 🙏

Comment (dsinghvi) [reply generators/python/sdk/CHANGELOG.md:10]:
```python
# Before
Color = typing.Literal["red", "blue"]

# After
Color = typing.Union[typing.AnyStr, typing.Literal["red", "blue"]]
```

Comment (armandobelardo) [reply generators/python/sdk/CHANGELOG.md]: updated!

**Comment by dsinghvi** [comment]
🚀

**Pull request merged**